lib/elixir/lib/kernel.ex

```diff
@@ -5753,6 +5753,69 @@ defmodule Kernel do
     end
   end
 
+  @doc """
+  Debugs the given `code`.
+
+  `dbg/2` can be used to debug the given `code` through a configurable debug function.
+
+  ## Configuring debug function
+
+  The debug function can be configured at compile time through the `:dbg_fun` key
+  of the `:elixir_application`. The debug function must be a `{module, function, args}`
+  tuple. The `function` function in `module` will be invoked with three arguments
+  *prepended* to `args`:
+
+    1. The AST of `code`
+    1. The AST of `options`
+    1. The `Macro.Env` environment of where `dbg/2` is invoked
```

Comment:
```suggestion
    2. The AST of `options`
    3. The `Macro.Env` environment of where `dbg/2` is invoked
```

We don't use the same number format in Elixir.